### DISCUSSION TITLE
Take screenshot of page as shown in a specific windows size, instead of the current browser window size

### DISCUSSION THREAD

**Discussion by GeeZuZz** [niklasvh/html2canvas#2765] [Q&A]
I'm using html2canvas to take screenshot to PDF/Print.

But if the browser window of the user is wide, the image will not fit on a printed page.

Is it possible to have html2canvas take a screenshot of how the page looks if the browser was a specific width? E.g on a responsive page, I want to take a screenshot of how the page looks in the mobile view, regardless of what view the user currently is shown. 

Setting "windowWidth" doesn't seem to do anything for me.

**Comment by EdByrnee** [comment]
> I'm using html2canvas to take screenshot to PDF/Print.
> 
> But if the browser window of the user is wide, the image will not fit on a printed page.
> 
> Is it possible to have html2canvas take a screenshot of how the page looks if the browser was a specific width? E.g on a responsive page, I want to take a screenshot of how the page looks in the mobile view, regardless of what view the user currently is shown.
> 
> Setting "windowWidth" doesn't seem to do anything for me.

Did you manage to find a solution for this?

**Reply by Luudoo** [reply]
Hi GeeZuZz

I have the same problem, generating PDF mixing tables and charts as images (html2canvas + jsPDF with autotable).
But what is really strange is that this option was working for me a few days ago, and no longer today.
I can't explain, but have printed an example with somes images using différent values for this option, and there was really different results !
Niklas, if you read us, do you have any idea ?

Thanks

**Reply by GeeZuZz** [reply]
@EdByrnee No I never found a solution... :-/

**Reply by Luudoo** [reply]
OK, found the reason why it worked before, it was just not the same version of html2canvas and it has been updated in my project when i fixed another problem. Came back to 1.3.2 instead of 1.4.1 and it works again !
I'll post an issue, i'm not able to find the change that is responsible for that.
